flutter/lib/app_constants.dart

```diff
@@ -110,5 +110,5 @@ class BackendId {
 }
 
 class BenchmarkSettings {
-  static const double maxDuration = 10 * 60; // 10 minutes
+  static const double maxDuration = 30 * 60; // 30 minutes
```

Comment:
@freedomtan @Mostelk Please confirm that we want this change.

Reply:
suggestion from @mohitmundhragithub: change this only for slow devices when submitting slow devices. Or we can make this configurable?

Let's do it like the `expected latency` (i.e., make it configurable in settings protobuf file).

for 3.1 branch: it should be fine to merge this.
for master branch: let's make it configurable.